src/_guides/language/language-tour.md

```diff
@@ -2973,28 +2962,27 @@ class Person {
 }
 
 // An implementation of the Person interface.
-class Imposter implements Person {
-  // We have to define this, but we don't use it.
-  final _name = "";
+class Impostor implements Person {
+  get _name => '';
```

Comment:
IMHO, this is the proper why to implement the Person interface `_name` element. `Person._name` is a final field, so the implicit interface contains only a getter named `_name`. (And this way it avoids the partly awkward comment of "We have to define this, but we don't use it."

Reply:
Sounds good. Also, thanks for fixing the spelling of impostor! Hard to believe it went unnoticed for so long...